doc_src/set_color.txt

```diff
@@ -2,36 +2,35 @@
 
 \subsection set_color-synopsis Synopsis
 \fish{synopsis}
-set_color [OPTIONS] [COLOR]
+`set_color` [OPTIONS] *VALUE*
```

Comment:
Whoops. Yes. In the rest of the manpage backticks caused <tt> in HTML or boldness in `man`, and asterisks caused italics in HTML and underlines in the manpage.

These are all nice behaviors and match other manpages on my system for non-fish projects where command names are bolded and keywords are underlined.